web/src/components/SongQueueItem.js

```diff
@@ -2,6 +2,34 @@ import React from 'react';
 import classNames from 'classnames';
 import {timeStr} from '../util/song';
 
+const QueueItemStatus = {
+  INVALID: 0,
+  UNKNOWN: 1,
+  WAITING: 2,
+  DOWNLOADING: 3,
+  PROCESSING: 4,
+  DONE: 5,
+};
+
+function setStatus(status) {

```

Comment:
This isn't named well because we're not setting anything. Maybe `getStatusText` or something.